Hunting Queries/ASimProcess/imProcess_ProcessEntropy.yaml

```diff
@@ -25,10 +24,14 @@ query: |
   let maxLimit = 6*24;
   let removeHigh = imProcessCreate 
   | where TimeGenerated >= ago(1d)
+  | extend TargetProcessFilePath = column_ifexists("TaregetProcessFilePath",TargetProcessName)
```

Comment:
imProcessCreate is a fixed schema and thus TargetProcessFilePath will never exist. extend TargetProcessFilePath to TargetProcessName only.
Moreover the spelling is incorrect so even if it was present, values will not get fetched.